lib/msal-core/src/error/ClientConfigurationError.ts

```diff
@@ -7,7 +7,12 @@ import { Constants } from "../utils/Constants";
 import { ClientAuthError } from "./ClientAuthError";
 import { TelemetryOptions } from "../Configuration";
 
-export const ClientConfigurationErrorMessage = {
+interface IClientConfigurationErrorMessage {
```

Comment:
Why are we making this change for error message classes, and should it be done for all error message classes?

Reply:
I felt we should codify the pattern of `code/desc` in the object below. I would actually complete refactor this into a different pattern, but felt this was an incremental improvement.